app/src/main/res/menu/bottom_navigation_constraint.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/action_compass"
+        android:enabled="true"
+        android:icon="@drawable/ic_lightbulb"
+        android:title="@string/constraint_menu_1"
+        app:showAsAction="ifRoom" />
```

Comment:
teraz masz tu ifRoom, wczesniej bylo never. Wydaje mi sie jakbys po prostu przekopiowywal to z jakiegos stackoverflow (nie mowie ze to zle, ale boje sie ze  mozesz robic to bezmyslnie). Sprawdz czym roznia sie never, ifRoom i chyba always? jest jeszcze jakas trzecia opcja